cmd/root.go

```diff
@@ -22,10 +22,10 @@ import (
 	jsonHandler "github.com/apex/log/handlers/json"
 	levelHandler "github.com/apex/log/handlers/level"
 	multiHandler "github.com/apex/log/handlers/multi"
+	"github.com/dotpy3/go-elastic"
```

Comment:
I don't really like this. Can't we wait for the PR to be merged? In the meantime, we can also just vendor from your fork.